articles/application-development/modelling/domain-designer/how-to-use-the-domain-designer/how-to-use-the-domain-designer.md

```diff
@@ -184,16 +180,6 @@ Now re-add the classes which were removed from the other Diagram, namely the fol
 
 Lay them out so that `Shape` is located center-top of the diagram, `Triangle` bottom-left and `Circle` bottom-right.
 
-> [!NOTE]
```

Comment:
This note mentions 4 associations, there will only be two at this stage only later in the steps will this apply for circle and triangle. The context of the note is correct when moved further down